accounts/accounts.go

```diff
@@ -90,24 +117,27 @@ func (auth *Authentication) Find(username string) *Account {
 	return nil
 }
 
+// TODO: Quick fix to expose the file name
+func (auth *Authentication) GetFilepath() string {
+	return auth.db.GetFilepath()
+}
+
 // Lock the underlying accounts file
-func (auth *Authentication) Lock() {
+func (auth *Authentication) file_lock() {
```

Comment:
We follow https://github.com/golang/go/wiki/CodeReviewComments, so in Go style all methods and variables are named using camelCase.